deployment/src/main/java/io/quarkiverse/fx/deployment/QuarkusFxExtensionProcessor.java

```diff
@@ -41,7 +42,7 @@ AdditionalBeanBuildItem fxmlLoader() {
 
     @BuildStep
     AdditionalBeanBuildItem primaryStage() {
-        return new AdditionalBeanBuildItem(PrimaryStage.class);
+        return new AdditionalBeanBuildItem(PrimaryStage.class, FxApplication.class);
```

Comment:
Either rename the method to indicate or make a dedicated `@BuildStep` for FxApplication

Reply:
Ok, I added a new method.